public/app/features/plugins/admin/hooks/usePluginInfo.tsx

```diff
@@ -21,7 +20,7 @@ export const usePluginInfo = (plugin?: CatalogPlugin): PageInfoItem[] => {
   const latestCompatibleVersion = getLatestCompatibleVersion(plugin.details?.versions);
   const version = plugin.installedVersion || latestCompatibleVersion?.version;
 
-  if (Boolean(version)) {
+  if (Boolean(version) && version !== '%VERSION%') {
```

Comment:
I think it's gonna become confusing why we actually have this condition here. I would either add some explanation above or a link to the github comment which gives some context (https://github.com/grafana/grafana/pull/75833/files/c97534d2fded29e7998d1fd3924837fef6b6057c#r1345834600).

Reply:
added a comment

Reply:
can't this be handled on the backend instead? when reading the plugin.json file detecting this and replacing it with the Grafana version?

Reply:
Agree with @torkelo. Feels like a better solution to fix it at the "core", if that would be possible.